resources/content/articles/en/hello-world-example-en.md

```diff
@@ -0,0 +1,89 @@
+# Creating, Deploying, and Testing a HelloWorld smart contract
+
+### Pre-requisites
+
+Deploying smart contracts on the K Ethereum Virtual Machine (KEVM) devnet and testing Mallet requires:
+
+1. Compiling the Solidity code to KEVM bytecode, which you can do using [solc](https://hub.docker.com/r/ethereum/solc).
+2. The installation of the [Docker Engine](https://docs.docker.com/engine/install/)
+
+## Creating the contract
+Follow these steps to create a *HelloWorld* smart contract.
+
+**1. Create a Solidity file**
+
+Create a `myContract.sol` file:
+```
+    cat << EOF >myContract.sol
+    // SPDX-License-Identifier: MIT
+    pragma solidity >=0.5.1 <0.9.0;
+    contract HelloWorld {
+      function helloWorld() external pure returns (string memory) {
+        return "Hello, World!";
+      }
+    }
+    EOF
+```
+
+**2. Compile with `solc`:**
+
+Use the Docker command to compile with `solc`: (Assumes Docker has been installed.)
+
+    `docker run --rm -v $(pwd):/sources ethereum/solc:0.5.1 -o /sources --bin --abi /sources/myContract.sol`
```

Comment:
Doesn't need backticks because indented, same for other cmds on this page.
Either remove backticks or remove the indentation.